alerter/build.gradle

```diff
@@ -6,14 +6,12 @@ plugins {
 apply plugin: 'com.android.library'
 apply plugin: 'maven'
 apply plugin: "maven-publish"
+apply plugin: 'com.tapadoo.android.sputnik'
```

Comment:
What happens if I want to update the rules just for this project? Can I accomplish that without having to update Sputnik? If so then we can leave it in, but I'm concerned with diminishing the extensibility of the project. I want this decoupled as much as possible. 

Also something to be aware of is that if we're accepting PRs for Sputnik, any changes will effect future projects, so we have to be very careful.